src/status_im2/contexts/wallet/events.cljs

```diff
@@ -231,18 +232,37 @@
 
 (rf/reg-event-fx :wallet/store-last-collectible-details store-last-collectible-details)
 
+(def collectible-data-types
+  {:unique-id        0
+   :header           1
+   :details          2
+   :community-header 3})
+
+(def fetch-type
+  {:never-fetch         0
+   :always-fetch        1
+   :fetch-if-not-cached 2
+   :fetch-if-cache-old  3})
+
+(def max-cache-age-seconds 3600)
+
 (rf/reg-event-fx
  :wallet/request-collectibles
  (fn [{:keys [db]} [{:keys [start-at-index new-request?]}]]
    (let [request-id          0
          collectibles-filter nil
+         data-type           (collectible-data-types :header)
+         fetch-criteria      {:fetch-type            (fetch-type :fetch-if-not-cached)
+                              :max-cache-age-seconds max-cache-age-seconds}
          request-params      [request-id
                               [(chain/chain-id db)]
                               (map :address (:profile/wallet-accounts db))
                               collectibles-filter
                               start-at-index
-                              collectibles-request-batch-size]]
-     {:json-rpc/call [{:method     "wallet_filterOwnedCollectiblesAsync"
+                              collectibles-request-batch-size
+                              data-type
+                              fetch-criteria]]
+     {:json-rpc/call [{:method     "wallet_getOwnedCollectiblesAsync"
```

Comment:
I've noticed we are having trouble following the re-frame guideline for using only `:db` and `:fx` in the returned map (https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#registering-event-handlers). A lot of events in the repo only dispatch one effect (plus maybe the `:db`) so maybe we should relax that guideline. I would prefer if we had only one way of doing something and following re-frame's best practices, but it's tedious to comment in PRs to reinforce this guideline, so realistically speaking we should consider relaxing that guideline if we can't properly enforce it.